packages/notification-center/src/hooks/notifications-provider.tsx

```diff
@@ -64,33 +64,39 @@ export function NotificationsProvider({ children }: { children: React.ReactNode
     messageId: string,
     actionButtonType: ButtonTypeEnum,
     status: MessageActionStatusEnum,
-    payload?: Record<string, unknown>,
-    storeId = 'default_store'
+    payload?: Record<string, unknown>
   ) {
     await api.updateAction(messageId, actionButtonType, status, payload);
 
-    notifications[storeId] = notifications[storeId].map((message) => {
-      if (message._id === messageId) {
-        message.cta.action.status = MessageActionStatusEnum.DONE;
-      }
+    for (const storeId in notifications) {
+      notifications[storeId] = notifications[storeId].map((message) => {
+        if (message._id === messageId) {
+          message.cta.action.status = MessageActionStatusEnum.DONE;
+        }
 
-      return message;
-    });
+        return message;
+      });
+    }
 
-    setNotifications(notifications);
+    setNotifications(Object.assign({}, notifications));
   }
 
   async function refetch(storeId = 'default_store') {
     await fetchPage(0, true, storeId);
   }
 
+  function disposeDefault() {
+    const storeId = 'default_store';
+    delete notifications[storeId];
+  }
```

Comment:
Could you elaborate on why we are disposing the default In this scenario?

Reply:
@scopsy Sure, and i will think maybe there is a better way to do it. but the motivation is the following.
if the user does not add tabs that contain storeId a default_store key record will be stored which is great however if the user does provide 'tabs' the 'default_store' will be created but never used. 
what do you think?